File: Python/appConfig.py
BROKER = "127.0.0.1"    # MQTT broker address
PORT = 9071             # Port
STORE_CODE = "0001"     # Store code
AP_ID = "003N"          # AP ID
USER_NAME = "test"      # User name
PASSWORD = "123456"     # Password

TOPIC_CONFIG = f"/estation/{AP_ID}/configure"
TOPIC_TASK_ESL = f"/estation/{AP_ID}/taskESL"
TOPIC_TASK_ESL2 = f"/estation/{AP_ID}/taskESL2"
TOPIC_TASK_DSL = f"/estation/{AP_ID}/taskDSL"
TOPIC_TASK_DSL2 = f"/estation/{AP_ID}/taskDSL2"
TOPIC_FIRMWARE = f"/estation/{AP_ID}/firmware"
TOPIC_OTA = f"/estation/{AP_ID}/ota"
TOPIC_CERT = f"/estation/{AP_ID}/cert"

TOPIC_INFOR = f"/estation/{AP_ID}/infor"
TOPIC_RESULT = f"/estation/{AP_ID}/result"
TOPIC_HEARTBEAT = f"/estation/{AP_ID}/heartbeat"
TOPIC_MESSAGE = f"/estation/{AP_ID}/message"

TAG_ID_LIST = "TagID.txt"

File: Java/src/main/java/com/estation/demo/AppConfig.java
package com.estation.demo;

public final class AppConfig {
    public static final String BROKER = "127.0.0.1";
    public static final int PORT = 9071;
    public static final String AP_ID = "00C8";
    public static final String USER_NAME = "test";
    public static final String PASSWORD = "123456";

    public static final String TOPIC_CONFIG = "/estation/" + AP_ID + "/configure";
    public static final String TOPIC_TASK_ESL = "/estation/" + AP_ID + "/taskESL";
    public static final String TOPIC_TASK_ESL2 = "/estation/" + AP_ID + "/taskESL2";
    public static final String TOPIC_TASK_DSL = "/estation/" + AP_ID + "/taskDSL";
    public static final String TOPIC_TASK_DSL2 = "/estation/" + AP_ID + "/taskDSL2";
    public static final String TOPIC_FIRMWARE = "/estation/" + AP_ID + "/firmware";
    public static final String TOPIC_OTA = "/estation/" + AP_ID + "/ota";

    public static final String TOPIC_INFOR = "/estation/" + AP_ID + "/infor";
    public static final String TOPIC_RESULT = "/estation/" + AP_ID + "/result";
    public static final String TOPIC_HEARTBEAT = "/estation/" + AP_ID + "/heartbeat";
    public static final String TOPIC_MESSAGE = "/estation/" + AP_ID + "/message";

    public static final String TAG_ID_LIST = "TagID.txt";

    private AppConfig() {
    }
}
